uint16 drive_position
uint16 steering_position
